tests/APIv0/SmokeTest.php

```diff
@@ -419,10 +518,12 @@ public function testDiscussionAddBookMark() {
      */
     public function testRemoveDiscussionBookMark() {
         $api = $this->api();
-        $api->setUser($this->getTestUser());
+        $admin =  $this->api()->queryUserKey('Admin', true);
+        //use admin user so flood control isn't triggered
```

Comment:
Since APIv2 tests are "internal" requests and APIv0 tests are not, you can't use the same methods. However, you can configure the flood control to be more permissive by using the "Vanilla.Discussion.SpamCount" and "Vanilla.Discussion.SpamTime" config keys.